packages/grafana-schema/src/common/table.cue

```diff
@@ -61,7 +70,7 @@ TableFieldOptions: {
 	align: FieldTextAlignment | *"auto"
 	// This field is deprecated in favor of using cellOptions
 	displayMode?: TableCellDisplayMode
-	cellOptions: TableCellOptions
+	cellOptions: TableCellOptions | *{ type: TableCellDisplayMode | *"auto" }
```

Comment:
This is an odd construct - are you trying to work around https://github.com/grafana/cuetsy/issues/74 ?

What i'd expect here is really this:
```cue
cellOptions: TableCellOptions | *{ type: TableCellDisplayMode & "auto" }
```

but that's what doesn't work, per that other issue